src/elm/Session/Guest.elm

```diff
@@ -86,101 +90,122 @@ view thisMsg page ({ shared } as model) content =
             Shared.viewFullError shared
                 err
                 ClickedTryAgainTranslation
-                "An error ocurred while loading translation."
+                "An error occurred while loading translation."
                 |> Html.map thisMsg
 
         _ ->
-            let
-                onClickCloseLanguageNav =
-                    if not model.showLanguageNav then
-                        onClick (ShowLanguageNav False)
-
-                    else
-                        style "" ""
-
-                currentYear : String
-                currentYear =
-                    Time.toYear Time.utc shared.now
-                        |> String.fromInt
-            in
             div
-                [ class "main-login-grid min-h-screen"
+                [ class "md:flex" ]
+                [ div
+                    -- Left part with background and quote (desktop only)
+                    [ class "hidden md:block md:visible min-h-screen md:w-2/5"
+                    , class "bg-center bg-no-repeat"
+                    , style "background-color" "#EFF9FB"
+                    , style "background-position" "center bottom"
+                    , style "background-size" "auto 80%"
+                    , style "background-image" "url(images/auth_bg_full.png)"
+                    ]
+                    [ viewQuote
+                        { photoSrc = "images/woman.png"
+                        , name = "Victoria Lane"
+                        , occupation = "Founder"
+                        , quote = "We are designing for a global user base, we can get feedback not just from drivers we see in San Francisco"
```

Comment:
We should use an actual quote here, from an actual user, using an actual photo lol

@heltonlr can you help us on that? It also need to be localised

Reply:
I've hidden the user quote until we have some actual user quotes. I will add them as a part of #238.